etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -24,9 +12,20 @@
 import java.util.List;
 import java.util.Properties;
 
+import org.apache.commons.dbutils.DbUtils;
+import org.apache.commons.dbutils.QueryRunner;
+import org.apache.commons.dbutils.ResultSetHandler;
+import org.apache.commons.dbutils.handlers.ScalarHandler;
+import org.apache.commons.lang.time.StopWatch;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.pih.hivmigration.etl.sql.util.FileParser;
+import org.pih.hivmigration.etl.sql.util.SqlStatementParser;
+import org.springframework.util.StringUtils;
+
 abstract class SqlMigrator {
 
-    private static final Log log = LogFactory.getLog(SqlMigrator.class);
+    private final Log log = LogFactory.getLog(getClass());
```

Comment:
FYI - this change makes it such that the log file shows the actual Migrator class that's running and spitting out messages, rather than always just showing SqlMigrator.  This makes it much easier to see where in the migration things are happening as you scroll / watch the log.